dev/src/Split/ReleaseNotes.php

```diff
@@ -69,7 +69,7 @@ private function parseChangelog($changelog)
             $nodeValue = trim($d->nodeValue);
 
             $firstLine = explode(PHP_EOL, $nodeValue)[0];
-            preg_match('/\S{0,}\/(\S{1,})/', $firstLine, $matches);
+            preg_match('/\S{0,}\/(\S{1,}):?/', $firstLine, $matches);
```

Comment:
This is why the release notes for individual packages have been broken for a year

Reply:
This will have the tests pass too:
```suggestion
            preg_match('/\S{0,}\/(\S{1,})\:?/', $firstLine, $matches);
```

Reply:
For some reason the question mark was what caused the regex to fail. I tested this on regex101.com first, but sometimes PHP's regex is divergent. I have reverted both these changes.